Tests/SpackSmokeTest/CMakeLists.txt

```diff
@@ -17,9 +17,20 @@ set(_src_dir   ${_base_dir}/Source)
 set(_input_dir ${_base_dir}/Exec)
 
 
+if (AMReX_FORTRAN_INTERFACES)
```

Comment:
Where is this variable set again for the smoke test?

Reply:
This variable isn't set in the smoke test. It was set with the original AMReX install. 

I put it there because without it, the smoke test fails when you install with `spack install amrex+fortran`.

Reply:
Got ya, I was just wondering where it gets injected :)

Reply:
In the Spack amrex package (amrex/package.py), in the self test section around line 294. The command `args.extend(self.cmake_args())` contains the  FORTRAN_INTERFACES flag and several others.

Reply:
Thanks!! :)